builder/proxmox/builder.go

```diff
@@ -152,16 +152,32 @@ func commHost(host string) func(state multistep.StateBag) (string, error) {
 // Reads the first non-loopback interface's IP address from the VM.
 // qemu-guest-agent package must be installed on the VM
 func getVMIP(state multistep.StateBag) (string, error) {
-	c := state.Get("proxmoxClient").(*proxmox.Client)
+	client := state.Get("proxmoxClient").(*proxmox.Client)
+	config := state.Get("Config").(*Config)
```

Comment:
I don't think this is case insensitive(?), so you'll need to use the same case as when `state.Put` is called above (all lowercase)